apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Component,
+  Input,
+  Output,
+  EventEmitter,
+  ViewChildren,
+  QueryList,
+  ElementRef,
+  AfterViewInit,
+} from '@angular/core';
+import { BaUxdNode, BaUxdEdge } from '@dynatrace/shared/barista-definitions';
+import { SafeHtml, DomSanitizer } from '@angular/platform-browser';
+import { cloneDeep } from 'lodash';
+
+@Component({
+  selector: 'ba-decision-graph-node',
+  templateUrl: './ba-decision-graph-node.html',
+  styleUrls: ['./ba-decision-graph-node.scss'],
+})
+export class BaDecisionGraphNode implements AfterViewInit {
+  @Input()
+  set node(node: BaUxdNode) {
+    this._node = node;
+    this.resetProgress();
+    this._decisionGraphSteps.push(cloneDeep(node));
+  }
+
+  /** Startnode picked from the user */
+  _node: BaUxdNode | undefined;
+
+  /** Array of all nodes and edges */
+  @Input()
+  decisionGraphData: BaUxdNode[] = [];
+
+  /** Emits when user wants to start over */
+  @Output()
+  startOver = new EventEmitter<void>();
+
+  /** List of elementrefs containing displayed nodes */
+  @ViewChildren('nodes') nodes: QueryList<ElementRef>;
```

Comment:
```suggestion
  @ViewChildren('nodes') nodes: QueryList<ElementRef<HTMLElement>>;
```

Then the typecast in line 67 is not needed